app/client-v2/apps/goat/app/[lng]/(dashboard)/content/MoreMenu.tsx

```diff
@@ -10,23 +10,16 @@ import { Icon, Text, IconButton } from "@p4b/ui/components/theme";
 import { makeStyles } from "@p4b/ui/lib/ThemeProvider";
 
 interface MoreMenuProps {
-  rowInfo: {
-    id: string;
-    name: React.ReactNode;
-    type: React.ReactNode;
-    modified: string;
-    size: string;
-    info: [];
-    path: [];
-  } | null;
+  rowInfo: object | null;
+  handleDeleteItem: (object) => void;
```

Comment:
I think we should aim to not use "object" types as the client doesn't build due to linting errors.